index.html

```diff
@@ -3211,7 +3211,22 @@ <h3>Reserved Extension Points</h3>
               <th>Description</th>
             </tr>
           </thead>
+
           <tbody>
+            <tr>
+              <td>`confidenceMethod`</td>
+              <td>
+A property used for specifying how a <a>verifier</a> can gain more confidence
+in, the presentation of the credential being associated with a particular
+<a>subject</a>. The associated vocabulary URL MUST be
+`https://www.w3.org/2018/credentials#confidenceMethod`.
```

Comment:
```suggestion
`https://www.w3.org/2018/credentials#confirmationMethod `.
```

Reply:
It would be good to not invent new security terminology that can so easily be confused with existing RFCs that doe exactly the same thing:

> By including a "cnf" (confirmation) claim in a JWT, the issuer of the
> JWT declares that the presenter possesses a particular key and that
> the recipient can cryptographically confirm that the presenter has
> possession of that key.
   
https://datatracker.ietf.org/doc/html/rfc7800#section-3

Reply:
Except that they *don't* do exactly the same thing. The `cnf` claim (and, I think, the related `confirmationMethod`) is specifically tied to key possession, while the `confidenceMethod` is explicitly broader and non-specific, though it might *include* key possession.

Reply:
@TallTed you are correct they don't do the same thing, and one of them is more dangerous because it has an unbound domain... https://en.wikipedia.org/wiki/Confidence_trick